index.html

```diff
@@ -0,0 +1,52 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>My Portfolio</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Crete+Round&family=Poppins&family=Roboto:wght@700&display=swap"
+      rel="stylesheet"
+    />
+    <link rel="stylesheet" href="styles.css" />
+    <script
+      src="https://kit.fontawesome.com/f1193e9575.js"
+      crossorigin="anonymous"
+    ></script>
+  </head>
+  <body>
+    <header class="container">
+      <nav>
+        <section>
+          <span><a href="#">My logo</a></span>
+          <i class="fa fa-bars"></i>
+        </section>
+      </nav>
+      <article>
+        <h1 class="introduction">Hey There. I'm Amanuel</h1>
+        <h2 class="job-title">I'm a Software Developer</h2>
+        <p class="description">
+          I can help you build a product , feature or website Look through some
+          of my work and experience! If you like what you see and have a project
+          you need coded, don't hestiate to contact me.
+        </p>
+        <section class="contact">
+          <a class="logo1" href="https://github.com/amexabee"
+            ><i class="fa fa-github"></i
+          ></a>
+          <a
+            class="logo2"
+            href="https://www.linkedin.com/in/amanuel-galema-56900024a/"
+            ><i class="fa fa-linkedin"></i
+          ></a>
+          <a class="logo3" href="https://twitter.com/fountfillwblood"
+            ><i class="fa fa-twitter"></i
+          ></a>
+        </section>
```

Comment:
- [ ] You seem to have missed a couple of icons (AngelList and Medium). To match the design, it is necessary for the project to have all 5 icons, please make sure all of them are showing on the page. You can also make sure the github logo you are using is the one provided by figma.

Your design:
![image](https://user-images.githubusercontent.com/99830261/192068919-9d61a149-7e68-4099-907c-12284267cae0.png)
Figma template:
![image](https://user-images.githubusercontent.com/99830261/192069001-c98186c3-4a21-4f34-8df0-972184bb002a.png)